app/views/courses/settings.html.erb

```diff
@@ -334,8 +334,8 @@
         </td>
       </tr>
       <tr>
-        <td class="form-label"><%= f.label :license, :en => "License" %></td>
-        <td colspan="3">
+        <td class="form-label"><label for="license"><%= before_label('license', %{License}) %></label></td>
```

Comment:
This for should be `course_license` so that the label associates with the proper control.